client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -323,6 +327,17 @@ public final void initMongoHandler() {
         }
     }
 
+    public final void initOpenSearchHandler() {
```

Comment:
it is fine in this PR. but, as this code seems very similar to what done for Mongo, maybe @jgaleotti  you can refactor it to avoid duplication? 
@mmiguerodriguez   maybe leave a `// TODO` comment stating this code likely needs refactoring with the Mongo one

Reply:
please leve a TODO and when the Redis and the OpenSearch implementations are done I will refactor them out.